src/ocf_cache_priv.h

```diff
@@ -164,7 +164,7 @@ static inline uint64_t ocf_get_cache_occupancy(ocf_cache_t cache)
 	ocf_core_t core;
 	ocf_core_id_t core_id;
 
-	for_each_core(cache, core, core_id)
+	for_each_core_all(cache, core, core_id)
```

Comment:
When a core is being removed all the associated cachelines are invalidated so occupancy stats has to match. Detach doesn't change value of `core->added` so both `for_each_core()` and `for_each_core_all()` macros won't skip detached core. 

`Imo it would be safer to not have any assumptions about data structures of invalid cores.`
I agree it would be safer but without these assumptions we won't be able to perform recovery during passive mode. Each recovered cache lines has to be assigned to a core. And since we don't have access to valid superblock before activate we have to assume that any core my be valid. Otherwise assigning cache lines to cores has to be done during  the activate which is time consuming.

Reply:
In standby mode the cumulative occupancy is required only for `ocf_lru_populate()` which uses this value only for sanity check, so this is not a strong dependency. How about disabling this sanity check in standby mode?

Reply:
I've added `safe` param to the `ocf_lru_populate()`. Let me know what do you think about the new way of populating the freelist during the standby-init-recovery